src/Evolution/Executables/GrMhd/GhValenciaDivClean/CMakeLists.txt

```diff
@@ -52,43 +52,57 @@ set(LIBS_TO_LINK
   ValenciaDivClean
   )
 
+set(LIBS_TO_LINK_WITH_CONTROL_SYSTEM
+    ControlSystem
+    ControlSystemMeasurements
+    ${LIBS_TO_LINK}
+)
+
 add_ghmhd_executable(
   ""
-  evolution::NumericInitialData
+  "evolution::NumericInitialData,false"
   "${LIBS_TO_LINK}"
   )
 
+add_ghmhd_executable(
+  "Bns"
+  "evolution::NumericInitialData,true"
+  "${LIBS_TO_LINK_WITH_CONTROL_SYSTEM}"
+  )
+
 if (TARGET SpEC::Exporter)
   target_compile_definitions(
     EvolveGhValenciaDivClean PRIVATE HAS_SPEC_EXPORTER)
+  target_compile_definitions(
+    EvolveGhValenciaDivCleanBns PRIVATE HAS_SPEC_EXPORTER)
 endif()
 
 add_ghmhd_with_horizon_executable(
   BondiHoyleAccretion
-  gh::Solutions::WrappedGr<grmhd::AnalyticData::BondiHoyleAccretion>
-  "${LIBS_TO_LINK}"
+  "gh::Solutions::WrappedGr<grmhd::AnalyticData::BondiHoyleAccretion>,false"
+  "${LIBS_TO_LINK};ApparentHorizons;ParallelInterpolation"
```

Comment:
Is the addition of ApparentHorizons supposed to be part of this commit?

Reply:
Yeah for some reason I got linking errors with these with my changes so I had to add them in.